client/my-sites/plan-features/header.jsx

```diff
@@ -293,12 +291,9 @@ export class PlanFeaturesHeader extends Component {
 			planMatches( planType, { group: GROUP_WPCOM, term: TERM_ANNUALLY } ) &&
 			relatedYearlyPlan
 		) {
-			if ( isBillingWordingExperiment ) {
-				return translate( 'billed as %(price)s annually', {
-					args: { price: relatedYearlyPlan.formatted_price },
-				} );
-			}
-			return translate( 'billed annually' );
+			return translate( 'billed as %(price)s annually', {
+				args: { price: relatedYearlyPlan.formatted_price },
+			} );
```

Comment:
@enejb Is this the right revert? Looking at the code from before, this string would be returned when `isBillingWordingExperiment` returned true.

Since the experiment is no longer running shouldn't we default to the false option which would be:

`return translate( 'billed annually' );` ?

Reply:
Yes, this is intentional. I think we should update the wording to match the desktop version and always showed the "billed as (price) annually' wording.

Reply:
the experiment was inconclusive so I think deploying either variation is good. In this case, I want to go with the one that matched desktop.